### DISCUSSION TITLE
Taxinomie

### DISCUSSION THREAD

**Discussion by ACTLEM** [cnumr/best-practices#55] [General]
- Définir la bonne norme ISO pour la catégorisation par étape de cycle de vie (Rémi L. propose celle qui pourrait correspondre le mieux)
- Définir les classifications et leurs segments/catégories: Profils (UX, Tech, PO, Support, ...) ou encore Couche Technique (Front, Back, Base de données, ...) ou ???

**Comment by Trankilloman** [comment]
Bonsoir bonsoir, j'ai testé, auprès de mes relations, le cycle de vie proposé par l'ISO 24748-1:2018 qui, pour rappel, propose les 6 étapes suivantes :
    Conception
    Développement
    Production
    Utilisation
    Support
    Fin de vie
Il semblerait que les 6 étapes soient plutôt pertinentes et leur nombre suffisant.
Il en ressort néanmoins qu'il serait intéressant de mettre des exemples pour chaque étape car chacune d'elles n'étaient par forcément très explicites selon l'interlocuteur (je pense notamment à la limite du périmètre entre développement, production et utilisation) qu'il s'agisse d'une personne qui va travailler from scratch, a partir d'un CMS et configurer, développer des plugins pour celui ci ou qui va "seulement" compléter des informations dessus.

**Comment by Trankilloman** [comment]
Concernant la classification par segment/catégorie ou cycle de vie, j'aime bien la démarche de la norme NF E 01-005 (qui s'applique aux produits mécaniques mais dont la logique est transposable a n'importe quel sujet).
Comme indiqué sur le lien suivant https://slideplayer.fr/slide/14321618/
la méthode consiste à poser des questions préliminaire pour ensuite s'orienter vers des étapes du cycle de vie pour ensuite se voir afficher des bonnes pratiques a analyser en fonction de la faisabilité stratégique et technique. Il serait donc possible de poser quelques questions au lecteur en préambule pour l'orienter vers tel ou tel groupe de BP. 
La sélection par étape du cycle de vie pourrait elle reprendre le fonctionnement de ecodesign pilot ou l'utilisateur est guidé vers des questions/BP en fonction de ce qu'il indique comme étape du cycle de vie qu'il estime avoir la possibilité de travailler.
